AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -1706,6 +1706,7 @@ armidaleexpress.com.au,bendigoadvertiser.com.au,canberratimes.com.au,centralwest
 ! Do not forget about '!#safari_cb_affinity' section for exclusions:
 ! LINK: AnnoyancesFilter/Popups/sections/antiadblock.txt#general
 !
+decompiler.com#%#//scriptlet('set-constant', 'baked32', 'true')
```

Comment:
Not an annoyance - it prevents to view the result.

Reply:
By exception, I mean unblocking a request or blocked item that is checked by the anti-adblock

Reply:
I checked in Safari on Mac/iOS - if works fine. But I agree about unblocking for `Content Blocker for Samsung/Yandex browsers` + iOS (free version does not support scriptlets)